cumulusci/cumulusci.yml

```
@@ -84,6 +84,10 @@ tasks:
             path: src
             revert_path: src.orig
         group: Salesforce Metadata
+    delete_data:
+        description: Delete matching records in an org
+        class_path: cumulusci.tasks.bulkdata.DeleteData
+        group: Test Data
```

```suggestion
    delete_dataset:
        description: Delete matching records in an org
        class_path: cumulusci.tasks.bulkdata.DeleteData
        group: 'Data Operations'
```

for consistency with the other bulkdata tasks

---

I agree with the changed group but I think that delete_data is a better name than delete_dataset.

This task is different than the _dataset tasks because it doesn't have anything to do with the files in the datasets/ directory.

---

I kind of buy that argument, but also think it's likely to contribute to people remembering it wrong and getting a "did you mean delete_data?" message. Any preference here, @davidmreed?

---

I think I'm with Paul on this.